File: baselibrary/src/main/java/com/yanxing/baselibrary/base/mvp/BaseView.java
package com.yanxing.baselibrary.base.mvp;

/**
 * @author 李双祥 on 2017/11/20.
 */
public interface BaseView<T> {

    /**
     * 请求的数据
     * @param data
     */
    void setData(T data);
}

File: baseactivitylibrary/src/main/java/com/yanxing/baselibrary/BaseView.kt
package com.yanxing.baselibrary

import com.trello.rxlifecycle2.LifecycleProvider
import com.trello.rxlifecycle2.android.ActivityEvent

/**
 * View接口的基类,LifecycleProvider 用于取消RxJava订阅
 * @author 李双祥 on 2018/7/5.
 */
interface BaseView : LifecycleProvider<ActivityEvent> {
}